README.md

```diff
@@ -502,6 +503,38 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation
+
+One or more requests may be cancelled during execution, for example, cancelled
```

Comment:
Can you improve this language? We should make clear distinction that this is client side request cancellation and not request being cancelled by the server.

How about changing `Request Cancellation` -> `Handling Client-Side Request Cancellation`?

Reply:
How about changing to `Request Cancellation Handling`, and have
```
One or more requests may be cancelled by the client during execution.
```
to follow immediately on the first line of the paragraph? Just to have a shorter title.